src/main/java/service/UserMapper.java

```diff
@@ -0,0 +1,15 @@
+package service;
+
+import dto.UserResponseDto;
+import java.util.List;
+import model.User;
+
+public interface UserMapper {
+    List<User> getUserListFromDto(List<UserResponseDto> userResponseDto);
+
+    User getUserFromDto(UserResponseDto userResponseDto);
+
+    List<UserResponseDto> getUserDtoListFromUser(List<User> users);
+
+    UserResponseDto getUserDtoFromUser(User user);
```

Comment:
Not sure about methods for lists, because as for me in this situation we skip SRC

Reply:
You're right, on the other hand if I remove this here I will be forced to do List<> logic in Controller but logic must be done in Services.
So I am not sure too😅.

Reply:
But as for mapping objects in dto is not business logic